sandbox-web/src/main/java/com/exadel/sandbox/controllers/location_controller/CountryController.java

```diff
@@ -41,19 +43,21 @@ public CountryController(CountryService countryService) {
             consumes = {"application/json"},
             path = "/newCountry")
```

Comment:
Instead of `/api/location/country/newCountry` is more canonical to use just POST `/api/location/country`